app/utils/sentry/index.js

```diff
@@ -45,27 +45,33 @@ function getDsn() {
 }
 
 export function captureException(error, logger, store) {
-    capture(() => {
-        Sentry.captureException(error, {logger});
-    }, store);
+    if (error && logger && store) {
```

Comment:
if one of this is null we won't be sending anything to sentry as it seems the library crashes here